benchmarks/perf-tool/sample-configs/filter-spec/filter-2-spec.json

```diff
@@ -0,0 +1,40 @@
+{
+    "bool":
+    {
+        "must":
+        [
+            {
+                "term":
+                {
+                    "taste": "salty"
+                }
+            },
+            {
+                "bool":
+                {
+                    "should":
+                    [
+                        {
+                            "bool":
```

Comment:
Why is this a nested bool? Why cant it be flat?

Reply:
not sure what format do you mean, can you please give an example

Reply:
I was thinking ORs can be represented with disjunctive max queries and ANDs can be represented with must or filter clauses. But, I think this will work as well.